cmd/controller/main.go

```diff
@@ -68,9 +73,12 @@ func main() {
 		"If set the metrics endpoint is served securely")
 	flag.BoolVar(&enableHTTP2, "enable-http2", false,
 		"If set, HTTP/2 will be enabled for the metrics and webhook servers")
-	opts := zap.Options{
-		Development: true,
-	}
+	//zap log config
+	flag.BoolVar(&development, "zap-devel", true, "Enable development mode")
```

Comment:
I think we can get away with dropping these extra commands and use the default zap arguments. Please see [above](https://github.com/openshift/instaslice-operator/pull/149/files#r1792319100)

Reply:
Did a bit of refactoring for the flags and gotten them out of the main. Please have a look one more time if you think they can be useful or not for https://github.com/openshift/instaslice-operator/issues/119